### ISSUE TITLE
Environment Configuration

### ISSUE THREAD

**Issue by XM-WANG** [ZikangZhou/HiVT#5]
Hi, I am still confused about the environment:

I follow the instructions:
```
conda create -n HiVT python=3.8
conda activate HiVT
conda install pytorch==1.8.0 cudatoolkit=11.1 -c pytorch -c conda-forge
conda install pytorch-geometric==1.7.2 -c rusty1s -c conda-forge
conda install pytorch-lightning==1.5.2 -c conda-forge
```
And get an error:
> ModuleNotFoundError: No module named 'torch_geometric.data.storage'

It seems that the PyG should be greater than 2.0.x (see this [link](https://github.com/pyg-team/pytorch_geometric/issues/3868))

However, after upgrading PyG to 2.0.1, a new error appeared:

> TypeError: __inc__() takes 3 positional arguments but 4 were given

Do you have any ideas? Many thanks!

**Comment by ZikangZhou** [comment]
Hello,

PyG 2 is somewhere incompatible with PyG 1 (see [PyG 2.0](https://github.com/pyg-team/pytorch_geometric/releases/tag/2.0.0)), while this repo uses PyG 1.7.2. Please make sure that you install 1.7.2.

If you use PyG 1.7.2, you need to import Dataloader from torch_geometric.data and I think you won't meet such an error.

**Comment by XM-WANG** [comment]
Sorry I didn't make it clear.

I indeed install those packages following the instructions in README. And then execute:
```
python train.py --root datasets/ --embed_dim 64
```
Then I got the error:
> ModuleNotFoundError: No module named 'torch_geometric.data.storage'

**Comment by XM-WANG** [comment]
![image](https://user-images.githubusercontent.com/43406982/175893377-3017b4a8-804c-4aca-b2d4-f7a8de3845b9.png)
I also verified in the IDEL, that the version is correct.

**Comment by XM-WANG** [comment]
Oh, I suppose I know why this error occurred.

As I met some other environment errors yesterday ([issue](https://github.com/ZikangZhou/HiVT/issues/4)), I fixed myself.

At that time I am not sure which part is wrong, so I changed many packages with different versions. And found that it is because of the wrong torch. (torch 1.8.1 with conda)

Then I ran `train.py` and it started to process the data and took around 12 hours. 

And today I met errors:

> TypeError: inc() takes 3 positional arguments but 4 were given

and re-install these packages.

However, those data have been processed with the wrong version (PyG 2.x) and didn't re-processed again, so that lead to the error:

> ModuleNotFoundError: No module named 'torch_geometric.data.storage'

I will delete all processed data and re-try again.

Sorry for bothering you again.
